sorted_words.rb

```diff
@@ -0,0 +1,30 @@
+def creating_array_of_words
+    array_of_words = []
+    @array_of_words = array_of_words
```

Comment:
Same as previous suggestion, you can just set the variable as an object when first declaring it. And change the rest of the variables along with it.
```suggestion
    @array_of_words = []
```